multiqc/modules/samtools/flagstat.py

```diff
@@ -51,7 +51,7 @@ def parse_samtools_flagstats(self):
             keys = OrderedDict()
             reads = {
                 'min': 0,
-                'modify': lambda x: float(x) / 1000000.0,
+                'modify': lambda x: float(x) * config.read_count_multiplier,
                 'suffix': 'M reads',
```

Comment:
Please can you use `config. read_count_prefix` here

Reply:
And please add `config.read_count_desc` for the ones that have descriptions too..

Reply:
`M reads` is short for `Million reads`. But it may not always be a million. `config.read_count_prefix` has either `M` or `K` or a blank string or whatever, which should correspond properly.

`read_count_desc` has the long form of this. See https://multiqc.info/docs/#number-base-multiplier